text/0000-custom-dst.md

```diff
@@ -86,35 +89,36 @@ struct Foo {
 }
 
 // size_of_val(&foo) returns size_of_header::<Foo>() + size_of_val(&foo.y)
-// same with align_of_val - simply `align_of_header::<Foo>()`
+// same with align_of_val -
+//   `max(align_of_header::<Foo>(), align_of_val(&foo.y))`
 ```
 
 
 # Reference-level explanation
 [reference-level-explanation]: #reference-level-explanation
 
-By language trait, we mean that `DynamicallySized` is a lang item.
+By language trait, we mean that `Pointee` and `Contiguous` are language items.
 
 In addition to the explanation given above,
 we will also introduce three functions into the standard library,
 in `core::raw`, which allow you to create and destructure these
-pointers to `DynamicallySized` types:
+pointers to `!Sized` types:
 
 ```rust
 mod core::raw {
-    pub fn from_raw_parts<T: ?Sized>(
+    pub fn from_raw_parts<T: ?Contiguous>(
```

Comment:
Err does this RFC now require #2255?

Reply:
I mean, maybe? It seems like a silly question. We've already got `?Sized`, and these are the natural extension of that.